packages/accordions/src/styled/accordion/StyledHeader.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.header';
+
+export interface IStyledHeader {
+  label?: string;
+  isFocused?: boolean;
+  isCompact?: boolean;
+  isExpanded?: boolean;
```

Comment:
Are `label`, `isCompact`,  and `isExpanded` needed here?

Reply:
👀 I missed these during clean up. These aren't needed.